lib/log/log.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright 2019 Gravitational, Inc.
```

Comment:
Is this a mostly copy-paste of logrus interface? If so, we should probably use their header.

Reply:
Technically, it's not a copy - it's been adopted to mirror the interface but does not duplicate the original - not sure this needs the license copy-paste.